build/debug.go

```diff
@@ -0,0 +1,14 @@
+package build
```

Comment:
I am not reviewing the whole PR, just focusing on this since I am coming from #525 

Nice! One thing that I would change if we go with this approach is that I'd probably like to embed conditionally, depending on the presence of a `debug` build flag perhaps that we can use also to disable completely the debug menu in the CLI as well so that any `debug` related stuff is stripped out from the production binary since debug is used for development purposes only.

How does it sound guys? @Olshansk @0xBigBoss

Reply:
I left another comment about this elsewhere, but I think we should use the runtime manager for this (see above)

@deblasis wdyt?

Reply:
@Olshansk I agree with you. Especially now with keybase and vault integration, we are totally relying on flags that in reality represent something that is kinda static and runtime/environment dependent.

Just using a random command as an example: 
![image](https://user-images.githubusercontent.com/29378614/221861554-9d073bf0-1e3a-4747-8c6d-4e10300ee934.png)

We don't want to expect the user to input all the time the `--keybase` and other flags if they use vault.

We want to offer the option to override that but I think we should reuse the pattern where we have defaults->config->overrides via environment variables/flags thanks to viper (12 factor app).